apps/studio/src/server/modules/site/site.service.ts

```diff
@@ -27,23 +27,35 @@ export const setSiteConfig = async (
 export const getNotification = async (siteId: number) => {
   const result = await db
     .selectFrom("Site")
-    .select(sql`config ->> 'notification'`.as("notification"))
+    .select((eb) =>
+      eb.ref("Site.config", "->").key("notification").as("notification"),
+    )
     .where("id", "=", siteId)
     .executeTakeFirstOrThrow()
 
-  return result.notification
+  // NOTE: Empty string denotes absence of notification on site.
+  return result.notification || ""
 }
 
 // TODO: Should triger immediate re-publish of site
+// NOTE: Empty string removes a notification
 export const setSiteNotification = async (
   siteId: number,
   notification: string,
 ) => {
-  return db
-    .updateTable("Site")
-    .set({
-      config: sql`jsonb_set(config, '{"notification"}', to_jsonb(${notification}::text))`,
-    })
-    .where("id", "=", siteId)
-    .executeTakeFirstOrThrow()
+  if (notification) {
```

Comment:
instead of handling a null string implicitly (users of the function might be unaware of this implicit handling), might we want to separate this function into its `setSiteNotification` and `clearSiteNotification` variants instead? Then we can have the same client and server schema, and the procedure handler calls the appropriate db service function.